arrr/__init__.py

```diff
@@ -35,55 +33,16 @@
 Documentation here: https://arrr.readthedocs.io/en/latest/
 """
 
-
 #: MAJOR, MINOR, RELEASE, STATUS [alpha, beta, final], VERSION
-_VERSION = (1, 0, 3)
-
+_VERSION = (1, 1, 0)
 
 #: Defines English to Pirate-ish word substitutions.
-_PIRATE_WORDS = {
-    "hello": "ahoy",
-    "hi": "arrr",
-    "my": "me",
-    "friend": "m'hearty",
-    "boy": "laddy",
-    "girl": "lassie",
-    "sir": "matey",
-    "miss": "proud beauty",
-    "stranger": "scurvy dog",
-    "boss": "foul blaggart",
-    "where": "whar",
-    "is": "be",
-    "the": "th'",
-    "you": "ye",
-    "old": "barnacle covered",
-    "happy": "grog-filled",
-    "nearby": "broadside",
-    "bathroom": "head",
-    "kitchen": "galley",
-    "pub": "fleabag inn",
-    "stop": "avast",
-    "yes": "aye",
-    "no": "nay",
-    "yay": "yo-ho-ho",
-    "money": "doubloons",
-    "treasure": "booty",
-    "strong": "heave-ho",
-    "take": "pillage",
-    "drink": "grog",
-    "idiot": "scallywag",
-    "sea": "briney deep",
-    "vote": "mutiny",
-    "song": "shanty",
-    "drunk": "three sheets to the wind",
-    "lol": "yo ho ho",
-    "talk": "parley",
-    "fail": "scupper",
-    "quickly": "smartly",
-    "captain": "cap'n",
-    "meeting": "parley with rum and cap'n",
-}
+_PIRATE_WORDS = dict()
+with open('BLABBER.csv') as nonsense:
```

Comment:
Python landlubbers think it good practiss to eschew side effects at importin’ time, like openin’ and readin’ file.
That’s fine thinkin’ but it requires ye to go roundabout and add some `init` function like this here: https://github.com/python/cpython/blob/main/Lib/mimetypes.py#L336-L337

So me wonder what cap’n be thinkin’.  Be fine to keep this simple if `arrr` be mainly a command-line program, not a module for other landlubbers to use.